### ISSUE TITLE
[feature request] Testing / stress-testing mode forcing rollout-model to always generate max_seqlen number of tokens

### ISSUE THREAD

**Issue by vadimkantorov** [verl-project/verl#2743]
This is useful to max-out VRAM usage, help testing for OOMs and tune the max_seqlen (to avoid non-deterministic OOMs in the future)

It would be okay if this mode forces the underlying LLM to produce gibberish (by ignoring EOS_ID until the very end)

**Comment by eric-haibin-lin** [comment]
Sounds reasonable. having a standalone script for this should help

**Comment by vadimkantorov** [comment]
Would be great at least supporting this as a config option for `rollout` - to make deterministic the produced number of tokens (and ignoring EOS_ID until the end) - hoping that vllm samplingparams / logits processors support this natively (in V1 this should also be available)

**Comment by vadimkantorov** [comment]
Apparently this is already possible vis SamplingParams for vllm:
- https://github.com/vllm-project/vllm/issues/21672#issuecomment-3126161887

So might be enough to make it into a Verl config option

**Comment by vadimkantorov** [comment]
We've tried the `SamplingParams(min_tokens=...)` in vllm, it seems not impacting anything, at least in V1 (might got broken during V0->V1 evolution?).

But `ignore_eos=True` seems working. But maybe since it might generate many EOS, the EOS tokens also need to be added to stop_tokens or be replaced by some other token

**Comment by TristanThrush** [comment]
I had the same issue with min_tokens not doing anything. I found that downgrading to vllm==0.8.0 works for me, at least. vllm==0.8.5.post1 does not work though. So def seems to be a vllm bug. I've been using an older version of verl for my project though (I just pinned it so I dont have to deal with breaking verl changes in my project: https://github.com/volcengine/verl/tree/106d33f9ec7e437d55904f3e3d2b20aded4c3146)

**Comment by vadimkantorov** [comment]
Have you tried vllm 0.10.0? it seems that `min_tokens` is working again on my end. Does 0.10.0 work for you on your end wrt `min_tokens`? 

One thing to note is that `min_tokens` also includes the prompt_tokens. So if prompt_tokens >=  min_tokens, even outputting one token would satisfy the `min_tokens` - quite unintuitive, but that's how vllm works :/

**Comment by TristanThrush** [comment]
oh interesting - i haven't tried 0.10.0

**Comment by Kirrito-k423** [comment]
### 中文
定时清理检查发现：此 issue 最近一次非当前运行账号的公开回复或引用时间是 `2025-08-18T23:07:14Z`（comment，来自 `TristanThrush`），相对于本轮运行日期 `2026-07-12` 已超过 6 个月。因此按社区 issue 定时清理标准关闭。

已检查证据：Latest non-self public reply/reference: `2025-08-18T23:07:14Z` (comment by `TristanThrush`).
Comments API: https://api.github.com/repos/verl-project/verl/issues/2743/comments
Timeline API: https://api.github.com/repos/verl-project/verl/issues/2743/timeline

如果作者或社区成员仍能提供新的复现、证据或需求背景，请在此 issue 下继续评论请求重开，或创建新的 issue 并链接到这里。

### English
Scheduled cleanup check found that the latest public reply or reference by an account other than the current automation account was `2025-08-18T23:07:14Z` (comment by `TristanThrush`), which is more than 6 months before this run date `2026-07-12`. Therefore this issue is being closed under the scheduled stale-issue cleanup rule.

Evidence checked: Latest non-self public reply/reference: `2025-08-18T23:07:14Z` (comment by `TristanThrush`).
Comments API: https://api.github.com/repos/verl-project/verl/issues/2743/comments
Timeline API: https://api.github.com/repos/verl-project/verl/issues/2743/timeline

If the author or community members can still provide a fresh reproduction, evidence, or requirement context, please comment here to request reopening, or open a new issue and link back to this one.

This response was generated by a scheduled Codex GPT5.5 issue processing task.
